README.md

```diff
@@ -143,3 +143,14 @@ and apply it on your Kubernetes environment:
 kubectl create -f pod-monitor.yaml
 ```
 Once the custom resource is created in the Kubernetes environment the operator metrics are ready to explore [http://localhost:3000/explore](http://localhost:3000/explore).
+
+# Savepoints
+Savepoints can be triggered manually by flipping the value of the boolean variable `savepointTriggerNonce` in the job specification:
+```yaml
+ job:
+    jarURI: local:///opt/flink/examples/streaming/StateMachineExample.jar
+    parallelism: 2
+    upgradeMode: savepoint
+    state: running
+    savepointTriggerNonce: false
```

Comment:
`savepointTriggerNonce` should be an integer type, otherwise it defeats the purpose

Reply:
ok I misunderstood it, i'll fix it then

Reply:
pushed it, @gyfora thx, makes more sense now